src/wallet/wallet.cpp

```diff
@@ -569,6 +569,9 @@ void CWallet::SyncMetaData(std::pair<TxSpends::iterator, TxSpends::iterator> ran
             copyFrom = &mapWallet[hash];
         }
     }
+
+    assert(copyFrom);
```

Comment:
From what I've seen there is at least one element in the range:
```cpp
TxSpends::iterator it = range.first;
const CWalletTx* copyFrom = mapWallet[it->second];
for (; it != range.second; ++i) {
    ...
```
Maybe this assert should come first:
```cpp
assert(range.first != range.second);
```